test_e2e/env_setup/duckdb/create-duckdb-schema.py

```diff
@@ -0,0 +1,13 @@
+import duckdb
+import sys
+from pathlib import Path
+
+if __name__ == "__main__":
+    connection = duckdb.connect(sys.argv[1])
```

Comment:
I'm getting an error here. Can duckdb really connect to a completely empty .db file. if so, cool!

`duckdb.duckdb.IOException: IO Error: The file "/Users/blabla/test_e2e/elements.db" exists, but it is not a valid DuckDB database file!`

Reply:
Looks like you just need to remove the `touch` in the other file.

Reply:
that makes sense. the duckdb client creates the database file when a database does not exists.